lib/devtools.js

```diff
@@ -0,0 +1,190 @@
+const fs = require('fs')
+const http = require('http')
+const { URL } = require('url')
+const cheerio = require('cheerio')
+const net = require('net')
+const path = require('path')
+
+const DevTools = function (config) {
+  this.config = config
+  this.values = null
+  this.mockBillValidatorClient = null
+  this.webpage = null
+  this.counter = 0
+}
+
+DevTools.prototype.disablePairing = function disablePairing () {
+  this.webpage('#token-input').attr('disabled', true).css('background-color', '#ffffff26')
+  this.webpage('#token-submit').attr('disabled', true).css('background-color', '#ffffff26')
```

Comment:
If you agree, I would remove css modification on server side

```suggestion
  this.webpage('#token-input').attr('disabled', true)
  this.webpage('#token-submit').attr('disabled', true)
```

we can put this in css

```
button:disabled, input:disabled {
  background-color: #fff2;
}
```